### PR TITLE
Applying Win11 Styles to TabControl, Frame, Page

### PR DESCRIPTION
**Pull request by harshit7962**
# Description
Adding Win11 Styles to the following controls:
1) TabControl
2) Page
3) Frame

## Testing
Build Pass.
Verified the visual appearance changes of controls in a sample application.

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/dotnet/wpf/pull/8579)

### CODE REVIEW

File: src/Microsoft.DotNet.Wpf/src/Themes/PresentationFramework.Win11/Styles/Frame.xaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+﻿<!--
```
Comment (anjali-wpf) [inline src/Microsoft.DotNet.Wpf/src/Themes/PresentationFramework.Win11/Styles/Frame.xaml:1]: Update header information in all files.

**Review by anjali-wpf — CHANGES_REQUESTED** [review]
LGTM.

File: src/Microsoft.DotNet.Wpf/src/Themes/PresentationFramework.Win11/Styles/TabControl.xaml [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+    This Source Code Form is subject to the terms of the MIT License.
+    If a copy of the MIT was not distributed with this file, You can obtain one at https://opensource.org/licenses/MIT.
+    Copyright (C) Leszek Pomianowski and WPF UI Contributors.
+    All Rights Reserved.
+-->
+
+<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml">
+
+    <Style TargetType="{x:Type TabControl}">
+        <Setter Property="Foreground" Value="{DynamicResource TabViewForeground}" />
```
Comment (lindexi) [inline src/Microsoft.DotNet.Wpf/src/Themes/PresentationFramework.Win11/Styles/TabControl.xaml:11]: I am concerned that a large number of DynamicResource declarations in this style might impact performance. Could we consider using StaticResource instead?

Comment (harshit7962) [reply src/Microsoft.DotNet.Wpf/src/Themes/PresentationFramework.Win11/Styles/TabControl.xaml:11]: The resources will change based on the theme, such as light, dark, or high contrast. Defining these resources as static will only bind them to one of these themes. We are still evaluating the performance impact of these changes. Do you have any suggestions to work around it?

**Comment by tbibre** [comment]
Hi,
I haven't checked this commit in details, but last time I checked, WPF UI TabControl style was not respecting TabStripPlacement : https://github.com/lepoco/wpfui/issues/214

**Comment by pchaurasia14** [comment]
@tbibre - Sure thing. We'll check it out

File: src/Microsoft.DotNet.Wpf/src/Themes/PresentationFramework.Win11/Styles/TabControl.xaml (hunk 2 above)
Comment (harshit7962) [reply src/Microsoft.DotNet.Wpf/src/Themes/PresentationFramework.Win11/Styles/TabControl.xaml:11]: Resolving this for now as the issue is listed. Will keep things updating there,
Issue available [here.](https://github.com/dotnet/wpf/issues/8599)

**Comment by harshit7962** [comment]
@tbibre - We have raised the issue [here](https://github.com/dotnet/wpf/issues/8606). Will keep updating leads there. Thanks for bringing this to our notice.

**Review by dipeshmsft — APPROVED** [review]

**Review by anjali-wpf — APPROVED** [review]

**Pull request merged**